### DISCUSSION TITLE
Networking and Asio migration

### DISCUSSION THREAD

**Discussion by pfeatherstone** [NVIDIA/stdexec#1133] [General]
Are there examples somewhere that show how to use stdexec for networking?
Will the networking model look similar to how Asio works?

**Comment by maikel** [comment]
Hi,

there is a paper of @dietmarkuehl especially about networking which is https://wg21.link/p2762

I am also very sure that Dietmar has wrapped lots of reactors. Not sure about Asios `io_context`, tho. He might dabble in here. I think there is some implementation to be found here https://github.com/dietmarkuehl/kuhllib

I've experimented myself with file i/o and networking with the `io_uring_context` in my own repository, but its largely undocumented and in some aspects quite different to what is proposed, here https://github.com/maikel/senders-io

HTH,
Maikel

**Comment by ericniebler** [comment]
there are now two(!) open PRs to integrate stdexec with asio:

* #1501 
* #1503
